CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.19.6]
+--------
+* feat: removed unencrypted columns of user data credentials from models in blackboard config ENT 8010
```

Comment:
can we remove ticket number from this

Reply:
nit: Remove client_id and client_secret columns from `BlackboardEnterpriseCustomerConfiguration`